sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/TransactionalBatchPatchItemRequestOptions.java

```diff
@@ -0,0 +1,59 @@
+package com.azure.cosmos;
+
+import com.azure.cosmos.implementation.RequestOptions;
+import com.azure.cosmos.util.Beta;
+
+@Beta(value = Beta.SinceVersion.V4_11_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
```

Comment:
Outdated version, see comment in BulkItemRequestOption regarding composition model